recipes-multimedia/alsa/alsa-utils_%.bbappend

```diff
@@ -0,0 +1,21 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+
+OVERRIDES_append = "${@bb.utils.contains("PACKAGECONFIG", "udev", ":udev-enabled", "",d)}"
+
+SRC_URI_append_hardkernel-odroidn2plus_udev-enabled = " file://soundconfig \
```

Comment:
It should work over most 64 amlogic socs, so you should use `meson-gx` instead of `hardkernel-odroidn2plus`

Reply:
Understood